tests/SBRPTests/SBRPFixture.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace SBRPTests;
+
+public class SBRPFixture
+{
+    public string WorkingDirectory { get; set; }
+
+    public SBRPFixture()
+    {
+        WorkingDirectory = Environment.CurrentDirectory = Environment.CurrentDirectory.Substring(0, Environment.CurrentDirectory.IndexOf("source-build-reference-packages") + 31);
```

Comment:
I prefer avoiding side effects from running the tests - e.g. Environment.CurrentDirectory.  This doesn't feel necessary. This is necessary if you ever want to run the tests in synchronously which is useful for performance reasons.

Reply:
Updated. Now it won't change Environment.CurrentDirectory.